src/secondary_operations/camera_to_robot_pose.py

```
@@ -0,0 +1,124 @@
+        Returns:
+            4x4 robot pose transform, or None if input is None.
+
+        Raises:
+            ValueError: If ``camera_pose`` is not a finite 4x4 matrix.
+        """
+        if camera_pose is None:
+            return None
+
+        camera_pose_matrix = np.asarray(camera_pose, dtype=float)
+        if camera_pose_matrix.shape != (4, 4) or not np.all(np.isfinite(camera_pose_matrix)):
+            raise ValueError("camera_pose must be a finite 4x4 matrix")
```

Ruff (0.15.0)

[warning] Avoid specifying long messages outside the exception class

(TRY003)